in_toto/models/statement.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python
+
+# Copyright New York University and the in-toto contributors
+# SPDX-License-Identifier: Apache-2.0
+
+"""
+<Program Name>
+  statement.py
+
+<Author>
+  Chasen Bettinger <bettingerchasen@gmail.com>
+
+<Started>
+  Mar 25, 2023
+
+<Copyright>
+  See LICENSE for licensing information.
+
+<Purpose>
+  Provides a class that represents a Statement, the middle layer of an 
+  attestation, binding it to a particular subject and unambiguously 
+  identifying the types of the Predicate as defined in the in-toto 
+  attestion spec.
+"""
+
+import attr
+import securesystemslib.formats
+from in_toto.models.common import Signable
+
+STATEMENT_TYPE = "https://in-toto.io/Statement/v1"
+FILENAME_FORMAT = "{step_name}.{keyid:.8}.link"
+FILENAME_FORMAT_SHORT = "{step_name}.link"
+UNFINISHED_FILENAME_FORMAT = ".{step_name}.{keyid:.8}.link-unfinished"
+UNFINISHED_FILENAME_FORMAT_GLOB = ".{step_name}.{pattern}.link-unfinished"
+
+
+@attr.s(repr=False, init=False)
+class Statement(Signable):
+  """The Statement is the middle layer of the attestation, binding it to a 
+  particular subject and unambiguously identifying the types of the Predicate.
+
+  A Link object is usually contained in a generic Metablock object for signing,
```

Comment:
I think we can do away with this and perhaps point to https://github.com/in-toto/attestation/blob/main/spec/v1.0/statement.md instead, especially as the descriptions below are from the spec.